integration_tests/sdk/preview_test.py

```diff
@@ -109,3 +111,47 @@ def bad_return():
 
     with pytest.raises(AqueductError):
         bad_return()
+
+
+@pytest.mark.enable_only_for_engine_type(ServiceType.K8S)
+def test_basic_get_kubernetes(client, data_integration, engine):
+    global_config({"engine": engine})
+    db = client.integration(data_integration)
+    sql_artifact = db.sql(query=SENTIMENT_SQL_QUERY)
+    sql_df = sql_artifact.get()
+    assert list(sql_df) == ["hotel_name", "review_date", "reviewer_nationality", "review"]
+    assert sql_df.shape[0] == 100
+
+    output_artifact = dummy_sentiment_model(sql_artifact)
+    assert output_artifact._dag.engine_config.type == RuntimeType.K8S
+    output_df = output_artifact.get()
+    assert list(output_df) == [
+        "hotel_name",
+        "review_date",
+        "reviewer_nationality",
+        "review",
+        "positivity",
+    ]
+    assert output_df.shape[0] == 100
+
+
+@pytest.mark.enable_only_for_engine_type(ServiceType.LAMBDA)
+def test_basic_get_lambda(client, data_integration, engine):
+    global_config({"engine": engine})
+    db = client.integration(data_integration)
+    sql_artifact = db.sql(query=SENTIMENT_SQL_QUERY)
+    sql_df = sql_artifact.get()
+    assert list(sql_df) == ["hotel_name", "review_date", "reviewer_nationality", "review"]
+    assert sql_df.shape[0] == 100
+
+    output_artifact = dummy_sentiment_model(sql_artifact)
+    assert output_artifact._dag.engine_config.type == RuntimeType.LAMBDA
+    output_df = output_artifact.get()
+    assert list(output_df) == [
+        "hotel_name",
+        "review_date",
+        "reviewer_nationality",
+        "review",
+        "positivity",
+    ]
+    assert output_df.shape[0] == 100
```

Comment:
Let's not duplicate the exact same test for different integrations. That's why we have this `enable_only_for_engine_types()` in the first place.

To get this merged into one test, just use:
`@pytest.mark.enable_only_for_engine_type(ServiceType.K8S, ServiceType.LAMBDA)`
And update the `enable_by_engine_type()` function in `conftest.py` to handle multiple engine types. Start by changing `enabled_engine_type = request.node.get_closest_marker("enable_only_for_engine_type").args[0]` to `enabled_engine_types = request.node.get_closest_marker("enable_only_for_engine_type").args` and dealing with the fallout in the rest of the method.

Let me know if you want to chat about this change, its pretty straightforward so I want you to take a crack at it.